llvm/lib/CodeGen/TargetInstrInfo.cpp

```diff
@@ -67,7 +70,9 @@ TargetInstrInfo::getRegClass(const MCInstrDesc &MCID, unsigned OpNum,
 /// point.
 void TargetInstrInfo::insertNoop(MachineBasicBlock &MBB,
                                  MachineBasicBlock::iterator MI) const {
-  llvm_unreachable("Target didn't implement insertNoop!");
+  // AIE Hack
+  return;
+//  llvm_unreachable("Target didn't implement insertNoop!");
```

Comment:
Don't you just need to implement this for your target?
I don't understand why we need this hack.

Reply:
I think this is technical debt and should be cleaned up.